test/smoke/appServers/global-resources/inheritedattributes_applicationinsights.json

```diff
@@ -0,0 +1,11 @@
+{
+  "connectionString": "InstrumentationKey=00000000-0000-0000-0000-0FEEDDADBEEF;IngestionEndpoint=http://host.docker.internal:6060/",
+  "preview": {
+    "inheritedAttributes": [
+      {
+        "key": "tenant",
```

Comment:
can you add tests for other data type supported? i am interested in seeing the structure of Array type, those a bit complicated ones.

Reply:
I think non-string attributes will be pretty rare for this feature

Reply:
if we expect string all the time, can you refactor SpanAttributeType and InheritedAttribute to support string only?

Reply:
I want to leave open the possibility for others

Reply:
at least remove the array types because Breeze won't accept array? we can add it back when it's supported? i just feel that this might never get used and become confusing later...

Reply:
arrays are converted to strings, see `Exporter.getStringValue()`